pom.xml

```diff
@@ -24,8 +24,7 @@
   </licenses>
   
   <properties>
-    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
-    <findbugs.failOnError>true</findbugs.failOnError>
+    <jenkins.version>1.566</jenkins.version>
   </properties>
```

Comment:
:bug: forgot

```xml
<java.level>6</java.level>
```

Wish we could default it based on `jenkins.version`, but for now it has to be set manually.